deployment/README.md

```diff
@@ -0,0 +1,37 @@
+# Deployment
+
+This document walks through instantiating an EC2 instance that will contain a
+copy of this repository, as well as a background daemon will update the OSMX
+database using the minutely diffs of `planet.osm` [provided by
+OpenStreetMap](https://planet.openstreetmap.org/replication/minute/).
+
+An AWS account with an existing EBS snapshot containing a `planet.osmx` database
+file is required.
+
+## Workflow
+
+The following AWS CLI command will instantiate the EC2 instance. You will have
+to substitute your own key name, security group, subnet ID, and snapshot ID.
+
+```bash
+aws ec2 run-instances \
+    --image-id ami-0bcc094591f354be2 \
+    --count 1 \
+    --instance-type t3.xlarge \
+    --key-name joker \
+    --security-group-ids sg-joker \
```

Comment:
Since this is going into docs for an open source project, it may be worth replacing "joker" with something more innocuous.